XRTK-Core/Packages/com.xrtk.core/Definitions/MixedRealityToolkitRootProfile.cs

```diff
@@ -20,12 +20,12 @@
 namespace XRTK.Definitions
 {
     /// <summary>
-    /// Configuration profile settings for the Mixed Reality Toolkit.
+    /// The root profile for the Mixed Reality Toolkit's settings.
     /// </summary>
-    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Toolkit Configuration Profile", fileName = "MixedRealityToolkitConfigurationProfile", order = (int)CreateProfileMenuItemIndices.Configuration)]
-    public class MixedRealityToolkitConfigurationProfile : BaseMixedRealityProfile
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Toolkit Root Profile", fileName = "MixedRealityToolkitRootProfile", order = (int)CreateProfileMenuItemIndices.Configuration)]
+    public sealed class MixedRealityToolkitRootProfile : BaseMixedRealityProfile
```

Comment:
Do we need the toolkit prefix?

Reply:
Well it's the root profile for the toolkit, so I think it makes sense in this case.